pkgs/by-name/in/installShellFiles/setup-hook.sh

```diff
@@ -218,18 +216,44 @@ installShellCompletion() {
             && cat "$arg" > "$outPath"
         else
             install -Dm644 -T "$arg" "$outPath"
-        fi || return
+        fi || \
+            { nixErrorLog "${FUNCNAME[0]}:" "install $arg to $outPath failed"; return 1; }
 
         if [ ! -s "$outPath" ]; then
-            echo "installShellCompletion: error: installed shell completion file \`$outPath' does not exist or has zero size" >&2
+            nixErrorLog "${FUNCNAME[0]}:" "installed shell completion file \`$outPath' does not exist or has zero size"
```

Comment:
Curious why not just the following:
```suggestion
            nixErrorLog "${FUNCNAME[0]}: installed shell completion file \`$outPath' does not exist or has zero size"
```

Reply:
`echoCmd` muscular memeory